history.txt

```diff
@@ -1,6 +1,41 @@
 OMERO version history
 =====================
 
+5.1.0 (April 2015)
+------------------
+
+A full, production-ready release of OMERO 5.1.0; updating the Data Model to
+the January 2015 schema, including support for units and new more
+flexible user-added metadata; and introducing new user features, new supported
+formats and many fixes and performance improvements:
+
+-  support for units throughout the Data Model allowing for example, pixel
+   sizes for electron microscopy to be stored in nanometers rather than being
+   set as micrometers
+-  new, searchable key-value pairs annotations for adding experimental
+   metadata (replacing OMERO.editor, which has been removed)
+-  improved workflow for rendering settings in the UI and parity between the
+   clients
+-  import images to OMERO from ImageJ and save ROIs and overlays from ImageJ
+   to OMERO
+-  importing as another user, previously only available for administrators, is
+   now usable by group owners as well, allowing you to import data that will
+   then be owned by the user you import it for
+-  improved performance for moving and deleting data
+-  import times are much improved for large datasets such as HCS and SPIM data
+-  improved performance for many file formats and new supported formats via
+   Bio-Formats (now over 140)
+-  improved rendering by removing calculation of PGI (Pretty Good Image) so
+   that default rendering settings are now set to global min/max if it is
+   calculated, or to full pixel range
```

Comment:
- removed the auto-levels calculation for initial rendering settings, simply using the min/max pixel intensities. (maybe omit the "if not calculated" bit here, since this is kinda an edge case and should probably be mentioned under performance improvements rather than rendering improvements.